village/src/App.js

```diff
@@ -1,15 +1,36 @@
 import React, { Component } from 'react';
+import axios from 'axios';
 
 import './App.css';
 import SmurfForm from './components/SmurfForm';
 import Smurfs from './components/Smurfs';
 
 class App extends Component {
+  state = {
+    smurfs: [],
+  };
+
+  componentDidMount(){
+    this.uploadedSmurfs();
+  }
+
+  uploadedSmurfs = () => {
+    axios.get('http://localhost:3333/smurfs')
+    .then(response => {
+      this.setState({
+        smurfs: response.data,
+      });
+    })
+    .catch((error) => {
+      console.error('Error try again...');
+    });
+  };
+
```

Comment:
I would recommend making the axios get request inside component did mount then pass data from the state via props.